core/templates/components/ratings/rating-display/rating-display.directive.spec.ts

```diff
@@ -75,20 +84,56 @@ describe('Rating display directive', function() {
     expect(ctrlScope.stars[3].cssClass).toBe('far fa-star');
     expect(ctrlScope.stars[4].cssClass).toBe('far fa-star');
 
-    ctrlScope.ratingValue = 2.25;
+    ctrlScope.ratingValue = undefined;
     outerScope.$digest();
-    expect(ctrlScope.stars[0].cssClass).toBe('fas fa-star');
-    expect(ctrlScope.stars[1].cssClass).toBe('fas fa-star');
-    expect(ctrlScope.stars[2].cssClass).toBe('far fa-star-half');
+    expect(ctrlScope.stars[0].cssClass).toBe('far fa-star');
+    expect(ctrlScope.stars[1].cssClass).toBe('far fa-star');
+    expect(ctrlScope.stars[2].cssClass).toBe('far fa-star');
     expect(ctrlScope.stars[3].cssClass).toBe('far fa-star');
     expect(ctrlScope.stars[4].cssClass).toBe('far fa-star');
 
+    ctrlScope.status = 'active';
     ctrlScope.ratingValue = 4.3;
     outerScope.$digest();
-    expect(ctrlScope.stars[0].cssClass).toBe('fas fa-star');
-    expect(ctrlScope.stars[1].cssClass).toBe('fas fa-star');
-    expect(ctrlScope.stars[2].cssClass).toBe('fas fa-star');
-    expect(ctrlScope.stars[3].cssClass).toBe('fas fa-star');
+    expect(ctrlScope.stars[0].cssClass)
```

Comment:
if lines 95 - 97 are the actions and the rest of the lines are the post-checks then insert a new line in between.

Reply:
Done